Cartfile

```diff
@@ -1,2 +1 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/search-core-sdk/MapboxCoreSearch.xcframework.json" == 2.2.0-rc.1
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 24.5.0-rc.1
```

Comment:
If I understand correctly, the Search SDK still depends on the MapboxCommon. Why the dependency was removed, and specifically why was it only removed from the Cartfile, but not from podspec and Package.swift?

It affects how the Search SDK is distributed through [Carthage](https://github.com/Carthage/Carthage)

Reply:
Ah I think I can keep Carthage support as-is and still achieve importing MapboxMaps through SPM in the MapboxSearch.xcodeproj. I'll push an update soon.